articles/post-knowledge_base.php

```diff
@@ -85,7 +85,12 @@
 		</div>
 	<?php endif; ?>
 
-	<?php comments_template(); ?>
+	<?php
+	// Display the comments template if the current user is logged in.
+	//if ( is_user_logged_in() ) {
```

Comment:
Indecisive? :trollface:

Reply:
Haha, yep! Reached a decision after discussing with @slocker3 and addressed this in another PR 🙂